staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server/server.go

```diff
@@ -25,7 +25,7 @@ import (
 	genericapiserver "k8s.io/apiserver/pkg/server"
 )
 
-func NewServerCommand(out, errOut io.Writer, stopCh <-chan struct{}) *cobra.Command {
+func newServerCommand(out, errOut io.Writer, stopCh <-chan struct{}) *cobra.Command {
```

Comment:
instead of lowercaseing the function add a comment on top to make golint happy.

the function is used from another package:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiextensions-apiserver/main.go#L35